### PR TITLE
Api 9 sql schema

### PR DESCRIPTION
**Pull request by dethe**
This is the client-side changes to match up with the server-side schema changes. Some items returned in the JSON are now singular instead of plural (or vice versa), have slightly different names, or are sub-objects of the main JSON object now.

### CODE REVIEW

**Comment by dethe** [comment]
@davidascher ready for review, will make a new PR for the next bit of API replacement. Should be faster, smaller chunks now.

Requires API PR 14.

File: src/utils/api.js [hunk 1]
```diff
@@ -91,6 +91,7 @@ class API {
       fetch(APIURL + '/case/' + caseId)
         .then(function (response) {
           response.json().then(function (json) {
+              console.log('in the heart of fetchCaseById, got the new case: %o', json.data);
```
Comment (davidascher) [inline src/utils/api.js]: remove stray log

File: src/utils/api.js [hunk 2]
```diff
@@ -131,7 +132,7 @@ class API {
   fetchNouns = function (noun) {
     return new Promise(function (resolve, reject) {
       try {
-        let url =APIURL + '/search/getAllForType?objType=' + noun 
+        let url =APIURL + '/search/getAllForType?objType=' + noun
```
Comment (davidascher) [inline src/utils/api.js]: may as well add a space before `APIURL`

File: src/utils/api.js (hunk 1 above)
Comment (dethe) [inline src/utils/api.js]: Agreed, thought I'd excised that already.

**Pull request merged**